File: JavaScript/Medium/Loop/Factorial.js
// Program to find factorial number

function factorial(number) {
    let fact = 1;
    for (let i = 1; i <= number; i++) {
        fact = fact * i;
    }

    return fact;
}
const fact = factorial(5);
console.log(`Factorial number is : ${fact}`);

File: C++/Medium/Cond-Loop/Factorial.cpp
// Program to find factorial number;

#include <iostream>
using namespace std;

int main() {
  int number;
  int fact = 1;

  cout << "Enter the number : ";
  cin >> number;

  for (int i=1; i<=number; i++) {
    fact = fact * i;
  }

  cout << "Factorial of " << number << " is : " << fact;
  return 0;
}